action.yml

```
@@ -23,6 +23,9 @@ inputs:
   force:
     description: 'Determines if force push is used'
     required: false
+  force_with_lease:
+    description: 'Determines if force-with-lease push is used'
+    required: false
```

Can you provide a default here 

```suggestion
    required: false
    default: false
```

---

Sure, but then it's not consistent with the rest.